src/rgw/rgw_op.cc

```diff
@@ -5800,6 +5915,7 @@ void RGWSetRequestPayment::execute(optional_yield y)
 
 int RGWInitMultipart::verify_permission(optional_yield y)
 {
+  //Not sure if adding tags here makes sense
```

Comment:
@mattbenjamin @adamemerson : Should I add code for using object tags for authorization, for multipart related object ops also?

Reply:
yes (and you did it ;)